src/Cache.cpp

```diff
@@ -210,7 +214,6 @@ Cache::setup()
                                 throw std::runtime_error(
                                   ("Unable to delete file " + file).toStdString().c_str());
                 }
-
                 env_.open(statePath.toStdString().c_str());
         }
 
```

Comment:
You need to now also patch the storeSecret, secret and removeSecret functions.

Reply:
I'll get to that on Monday.

Reply:
Sure, take your time :3